### PR TITLE
Submission times

### PR DESCRIPTION
**Pull request by phlax**
move suggestion saving to after unit to ensure consistent times

### CODE REVIEW

File: pootle/apps/pootle_store/utils.py [hunk 1]
```diff
@@ -212,17 +203,26 @@ def accept_suggestion(self, suggestion, update_unit):
         unit.target = suggestion.target
         if unit.state == FUZZY:
             unit.state = TRANSLATED
-
+        unit.submitted_by = suggestion.user
+        unit.submitted_on = current_time
+        unit.reviewed_by = self.reviewer
+        unit.reviewed_on = unit.submitted_on
+        unit.save(
+            submitted_by=suggestion.user,
```
Comment (unho) [inline pootle/apps/pootle_store/utils.py]: This was already set a few lines above, is it necessary to pass it to `save()`?

File: pootle/apps/pootle_store/utils.py [hunk 2]
```diff
@@ -212,17 +203,26 @@ def accept_suggestion(self, suggestion, update_unit):
...
             create_subs[SubmissionFields.STATE] = [old_state, unit.state]
         subs_created = []
         for field in create_subs:
             kwargs = {
-                'creation_time': current_time,
-                'translation_project': translation_project,
-                'submitter': self.reviewer,
+                'creation_time': unit.change.reviewed_on,
+                'translation_project': unit.store.translation_project,
+                'submitter': suggestion.user,
```
Comment (unho) [inline pootle/apps/pootle_store/utils.py]: Submissions for accepting suggestions were previously credited to the reviewer, but now to the suggester. Is that ok?

File: pootle/apps/pootle_store/utils.py (hunk 1 above)
Comment (phlax) [inline pootle/apps/pootle_store/utils.py]: yes - because its used to set unit change/unit_source objects - the above lines will be removed pretty soon

Comment (unho) [inline pootle/apps/pootle_store/utils.py]: As long as this code is simplified as much as possible, I am happy.

**Pull request merged**